integration/integration_test.go

```diff
@@ -221,6 +231,125 @@ func call[Resp any](module, verb string, req obj, onResponse func(t testing.TB,
 	}
 }
 
+func setUpKotlinModuleDb(dir string) assertion {
```

Comment:
Let's reuse the existing FTL test DB container on port 54320 rather than spinning up a new container.